src/main/webapp/app/programming/manage/exercise/programming-exercise.component.html

```
@@ -59,32 +45,81 @@
                             } @else {
                                 {{ programmingExercise.title }}
                             }
+                            @if (programmingExercise.shortName) {
+                                <br />
+                                <span class="badge bg-light text-dark border small"
+                                    >{{ programmingExercise.shortName | slice: 0 : 15 }}{{ (programmingExercise.shortName?.length ?? 0) > 15 ? '...' : '' }}</span
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)